packages/core/src/compiler/Style.ts

```diff
@@ -397,9 +399,76 @@ const checkDeclPatternsAndMakeEnv = (
   );
 };
 
+/**
+ * Helper fxn for checking well-formedness of selectors with predicate aliases
+ * @param m : Map<string, any> (only the keys of the map are used)
+ */
+const getKeyWordsFromMap = (m: any): string[] => {
```

Comment:
The short hand for getting an array of keys of a `Map`: 
```ts
const keywords: string[] = [...m.keys()];
```

Reply:
Since the alternative is short, I guess this function is not needed. Just map over the kw maps and do `m => [...m.keys()]`